galaxykit/command.py

```diff
@@ -454,12 +461,28 @@ def main():
                 resp = users.get_user_list(client)
                 print(format_list(resp["data"], "username"))
             elif args.operation == "create":
-                username, password = args.username, args.password
+                username, password, email, first_name, is_superuser, last_name = (
+                    args.username,
+                    args.password,
+                    args.email,
+                    args.first_name,
+                    args.is_superuser,
+                    args.last_name,
+                )
                 created, resp = users.get_or_create_user(
-                    client, username, password, None
+                    client,
+                    username,
+                    password,
+                    first_name,
+                    last_name,
+                    email,
+                    is_superuser,
```

Comment:
```suggestion
                    client,
                    username,
                    password,
                    None,
                    first_name,
                    last_name,
                    email,
                    is_superuser,
```

the 4th arg is still the group, not user names